shared/chat/conversation/bottom-banner/index.tsx

```diff
@@ -72,10 +77,72 @@ const BrokenTrackerBanner = ({users, onClick}: BrokenTrackerProps) =>
     </BannerBox>
   )
 
-const InviteBanner = ({users}: InviteProps) => (
-  <BannerBox color={globalColors.blue}>
-    <BannerText>Your messages to {users.join(' & ')} will unlock when they join Keybase.</BannerText>
-  </BannerBox>
-)
+const InviteBanner = ({users, openSMS, openShareSheet}: InviteProps) => {
+  if (!flags.sbsContacts) {
+    return (
+      <BannerBox color={globalColors.blue}>
+        <BannerText>Your messages to {users.join(' & ')} will unlock when they join Keybase.</BannerText>
+      </BannerBox>
+    )
+  }
+
+  // On mobile, single recipient, a phone number
+  if (isMobile && users.length === 1 && users[0].endsWith('@phone')) {
+    return (
+      <BannerBox color={globalColors.blue}>
+        <BannerText>Last step: summon Firstname Lastman!</BannerText>
```

Comment:
`// TODO Y2K-203 get the real full name`

What should we do if there is no full name? "Last step: summon them!" ? cc @cecileboucheron